sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Window.java

```diff
@@ -224,6 +224,22 @@
   }
 
   /**
+   * Override the amount of lateness allowed for data elements in the pipeline. Like
```

Comment:
elements in the output PCollection, and downstream until it is explicitly set again